### PR TITLE
DOMA-3884 linter rules improved

### PR DESCRIPTION
**Pull request by AleX83Xpert**
We've decided to use single quotes everywhere. This PR includes linter config improvements and all code fixed by linter with new rules applied.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -136,6 +137,7 @@
       "react/prop-types": "off",
       "react/no-children-prop": "off",
       "react/display-name": "warn",
+      "react/jsx-curly-brace-presence": "error",
```
Comment (sitozzz) [inline package.json:140]: I think there would be better to add more complex rule 

```suggestion
      "react/jsx-curly-brace-presence": ["error", { "props": "never", "children": "never" }],
```

**Comment by sitozzz** [comment]
With `error` level you should run 
```bash
eslint . --ext .js,.jsx,.ts,.tsx --fix
```

File: package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
         }
       ],
       "quotes": "off",
+      "jsx-quotes": ["warn", "prefer-double"],
```
Comment (pahaz) [inline package.json:78]: prefer-single

Comment (AleX83Xpert) [reply package.json:78]: Single is 
image

**Comment by AleX83Xpert** [comment]
> With `error` level you should run
> 
> ```shell
> eslint . --ext .js,.jsx,.ts,.tsx --fix
> ```

Yes. The "warn" level was set.

File: package.json (hunk 1 above)
Comment (AleX83Xpert) [reply package.json]: Done

**Comment by sitozzz** [comment]
> > With `error` level you should run
> > ```shell
> > eslint . --ext .js,.jsx,.ts,.tsx --fix
> > ```
> 
> Yes. The "warn" level was set.

I think it should be `error` level here. Otherwise no one will pay attention to it

image

**Comment by AleX83Xpert** [comment]
> > > With `error` level you should run
> > > ```shell
> > > eslint . --ext .js,.jsx,.ts,.tsx --fix
> > > ```
> > 
> > 
> > Yes. The "warn" level was set.
> 
> I think it should be `error` level here. Otherwise no one will pay attention to it
> 
> image

ok, I'll fix warnings after decision is made

**Review by pahaz — APPROVED** [review]

**Pull request merged**